cilium-cli.yaml

```
@@ -0,0 +1,51 @@
+package:
+  name: cilium-cli
+  version: 0.13.0
+  epoch: 0
+  description: CLI to install, manage & troubleshoot Kubernetes clusters running Cilium
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - gcc
+      - glibc-dev
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cilium/cilium-cli
+      tag: v${{package.version}}
+      destination: cilium-cli
+      expected-commit: ac2eca8005b7bdd677272845b5d0a6261e0e8a37
+  - assertions:
+      required-steps: 1
+    pipeline:
+      - if: ${{build.arch}} == 'x86_64'
+        runs: |
+          cd cilium-cli
+          export VERSION=$(git describe --tags --always)
+          mkdir -p release
+          GOARCH=amd64 CGO_ENABLED=0 go build \
+            -ldflags "-w -s -X 'github.com/cilium/cilium-cli/internal/cli/cmd.Version=${VERSION}'" \
+            -o release/cilium ./cmd/cilium
+          install -m755 -D release/cilium "${{targets.destdir}}"/usr/bin/cilium-cli
+      - if: ${{build.arch}} == 'aarch64'
+        runs: |
+          cd cilium-cli
+          export VERSION=$(git describe --tags --always)
+          mkdir -p release
+          GOARCH=amd64 CGO_ENABLED=0 go build \

```

```suggestion
          GOARCH=arm64 CGO_ENABLED=0 go build \

```

More generally, does this really need arch-specific builds? The [Makefile for Cilium CLI](https://github.com/cilium/cilium-cli/blob/master/Makefile) doesn’t have them

---

thanks @lizrice, I forgot the replace the GOARCH for the ARM build. 

> The [Makefile for Cilium CLI](https://github.com/cilium/cilium-cli/blob/master/Makefile?rgh-link-date=2023-02-28T15%3A58%3A20Z) doesn’t have them

https://github.com/cilium/cilium-cli/blob/25031f4bafb4d15e100d3a0fc3f66ad8d210a4a6/Makefile#L44, it seems that it has 🤔

---

You're right on that, I guess I was looking at an old version, sorry!

---

no worries, thanks for helping me!

---

Should this just run `make cilium` instead of duplicating its behavior here? If the Cilium project changes their Makefile I'd rather just get those updates for free rather than have to keep these in sync.

---

we tried to use `make cilium but it didn't work as we expected due to the 32-bit arch build, you can find these details in the below comments on why we switched from `make` to the present change.